### DISCUSSION TITLE
Output not generated

### DISCUSSION THREAD

**Discussion by vu5** [adithya-badidey/tiddlyjam#6] [Q&A] [answered]
I use a single file wiki for my note taking. It has some 950+ notes. I converted a copy of it to nodejs tiddlywiki (5.1.23) to make a static copy using tiddlyjam.

I have the following files in the output folder:

1. 404.html
2. blog.html
3. index.html
4. sitemap.xml

The index.html file has the name of the Wiki and the summary I put in the Welcome to TiddlyJam tiddler.

I can't see my other tiddlers anywhere. Can you tell me where I maybe going wrong? Do I need to tag the notes I need to export with some particular tags?

If you need more information please let me know.

**Comment by vu5** [comment, marked as answer]
I figured out. The pages that need to be published need to be tagged with $:/plugins/adithyab/tiddlyjam/page and later publish from the dashboard. Could have been helpful if you added this in the getting started page. Thank you for the great work

**Reply by adithya-badidey** [reply]
Hi @vu5, 
Awesome that you figured that part out yourself!

Thanks for trying out TiddlyJam and sharing your feedback. I will add more documentation and directions soon.

Adithya

**Reply by adithya-badidey** [reply]
I added some directions at https://tiddlyjam.com/customizing-your-tiddlyjam-website.html

Could you review it and let me know if it makes sense to you?

> I figured out. The pages that need to be published need to be tagged with $:/plugins/adithyab/tiddlyjam/page and later publish from the dashboard. Could have been helpful if you added this in the getting started page. Thank you for the great work

**Comment by vu5** [comment]
It is now followable for someone who is new to it. Thank you.

One other step I was stumbling on: index.html was not getting created. My
notes had a tiddler named "index". I think it is not common but in my case
it prevented the index.html file from being written. Once I renamed my
tiddler to something else it worked. You could consider adding that as a
heads-up.

**Comment by KTallguy** [comment]
@adithya-badidey 

Thanks for making TiddlyJam. I really like how you can export it to a static website. It feels very clean.
I'm an amateur at this stuff so apologies in advance, but I have a few questions about the export process as it is not working as I expect.

1. 
>The output folder should contain the html files. It will be rendered using the default theme - the same one as https://tiddlyjam.com/

When you say default theme, do you mean the one here? -> tiddlyjam.min.css
So just to be clear, this theme is different from the notebook theme that is used during editing?
If I wanted to do a dark theme closer to your website (https://adithyab.in/) should I edit the above css file?
I did look up bulma.css ... any basic tips on editing the colors or switching to the "dark" preset? 

2. 
I noticed that on the "home" button on individual blogs is not exporting the chevron button.
<i class="fas fa-chevron-circle-left" aria-hidden="true"></i>
Not sure if intentional.
![chrome_cnqIUqNw9t](https://user-images.githubusercontent.com/9643699/160238480-e368c76b-3d60-4749-8b43-3848e68bb9d7.png)

3.
I noticed that the font is different on the top left TiddlyJam text... any tips on fixing this?

4. I have other plugins like "appear" that hide and show pages. These don't look like they're exported properly. I should assume these Javascript things won't work?

5. Finally, if I have files like images I need to manually move the files folder... is that correct? 

Thanks and again sorry for the very amateur questions :)

**Comment by adithya-badidey** [comment]
1. For getting a dark theme, the easiest way would be to pick one from https://jenil.github.io/bulmaswatch/, download and replace the tiddlyjam.min.css with that one. That should do the trick (if it doesn't let me know; I'll try it and write a more detailed walk-through).
2. The chevron is rendered using the font-awesome library. My guess is that their registered api call is restricting the website it can originate from. Let me explore this a bit more.
3. I'm not sure why this is happenning. Could you share more about how I can reproduce this problem?
4. Tiddlywiki Javascript plugins will not work by default on the exported files (they can be made to work its a lot of work)
5. I use the [build-tw](https://github.com/adithya-badidey/tiddlyjam/blob/master/build-tw) script to export the pages and copy over the relevant files from the files folder. You can modify and use as you see fit.

Let me know in case of other questions. I am happy to help :)

**Comment by KTallguy** [comment]
@adithya-badidey 
Thank you so much for your answers, especially about the CSS. After I wrote you I was able to stumble through building bulma and tweaking some colors, but your link looks great.
Understood about Javascript plugins, etc. not working.
I'll explore a bit more about #3 when I can.
Best :)
